taskvine/src/tools/vine_plot_txn_log

```diff
@@ -145,7 +145,7 @@ def parse_tasks(log):
                 manager_info["Duties Sent"].append(time - manager_info["Start"])
```

Comment:
David, I suggest to change the transaction line for duties. "SEND" sounds like an event to me. E.g:

```
# subject target event arguments
DUTY dutyid event *args
```
It looks to me that you just need to swap target and event.